### ISSUE TITLE
Finetuning Espaloma guide?

### ISSUE THREAD

**Issue by jarvist** [choderalab/espaloma#199]
Is there a guide, or any additional information, on fine-tuning Espaloma? (i.e. taking the general model, and then supplementing with some molecule-specific electronic structure calculations, retraining with early stopping or regularisation)

Similarly, is there any documentation on how the main release models are trained? I note in the release notes `A manuscript describing the complete fitting process and assessment is forthcoming` - but is there anything to go on while that is in preparation? 
I very quickly get lost amongst the QCArchive / QCFractal etc.

**Comment by ijpulidos** [comment]
@jarvist Thanks for your interest in our tools. You are correct, we should improve the documentation and discoverability of how we are doing these things. We appreciate you bringing this to our attention.

While not perfect, there's a good amount of documentation on how to train and evaluate espaloma from https://github.com/choderalab/refit-espaloma (this is for the latest espaloma 0.3.1 and 0.3.2 models), I hope you find this helpful.

**Comment by jarvist** [comment]
Thanks! Yes, that looks very useful. 
I think the QCArchive might be a bit of a hurdle to overcome: I imagine it would be a lot more transparent to see the actual DFT calculations, similarly this would be necessary to do any active learning (& validation at the same step).

**Comment by JonathanHungerland** [comment]
@jarvist I was looking for the very same thing and started to go through the guide. Did you succeed in fine-tuning towards a molecule of yours?

**Comment by jarvist** [comment]
No, not yet - though we've been slightly distracted by the ease of
fine tuning MACE for more materials science orientated applications.
